pyqtgraph/graphicsItems/GridItem.py

```diff
@@ -26,6 +26,30 @@ def __init__(self, pen='default', textPen='default'):
         self.setPen(pen)
         self.setTextPen(textPen)
         self.setTickSpacing(x=[None, None, None], y=[None, None, None])
+        self.axes = None
+        self.ctrl = None
+
+
+    def setAxes(self, axes):
```

Comment:
if we're going to have a setter, to match the Qt API, we should have a getter method, `def axes(self)` which means we'll have to rename the attribute `self.axis` to `self._axis` or something like that.

Reply:
Also added the getter method `axes`, internal attribute changed to `self._axes` with commit e88d5ab